dev/fattest.simplicity/src/com/ibm/ws/fat/util/BuildShrinkWrap.java

```diff
@@ -0,0 +1,12 @@
+package com.ibm.ws.fat.util;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+public @interface BuildShrinkWrap {
```

Comment:
This looks like an interesting annotation, I don't really understand what it does though.  Could you explain a bit?  What is the difference between this annotation and just building the app in the `@BeforeClass`?